cassiopeia.go

```diff
@@ -6,18 +7,30 @@ import (
 	"regexp"
 )
 
+// I guest this is the main package of this service right?
+// so, every interface needed in this package should be declared here(e.g blogger and halloffame)
```

Comment:
It is, for some reason I was separating blogger and halloffame into some files is for the sake of clarity. When I was working on a big project, the package was gone into a big mess just only in a single file.

I accepted your request in this matter, however seems to be needing an elaborated reason(s) (e.g. web article, yt video, etc).

Reply:
https://medium.com/inside-bukalapak/how-i-design-my-code-54bfeb46d481
https://medium.com/@benbjohnson/structuring-applications-in-go-3b04be4ff091
https://stackoverflow.com/questions/14867452/what-is-a-sensible-way-to-layout-a-go-project
https://www.youtube.com/watch?v=cAWlv2SeQus

Reply:
> It is, for some reason I was separating blogger and halloffame into some files is for the sake of clarity. When I was working on a big project, the package was gone into a big mess just only in a single file.
> 
> I accepted your request in this matter, however seems to be needing an elaborated reason(s) (e.g. web article, yt video, etc).

I actually do this in two different way, 
1. put the interface where it will be used, or
2. Create another package and put the interface there and implement there also.

the second option will make it much more tidier if it is a big project
 example is like this https://github.com/RelieveID/camar

but it is the better to put interface in the file where it will be used,

Reply:
I think it is decided then, do the second.